### PR TITLE
Staking interface router ownership transfer

### PR DESCRIPTION
**Pull request by KPrasch**

### CODE REVIEW

File: tests/acceptance/cli/deploy/test_deploy_cli_commands.py [hunk 1]
```diff
@@ -160,6 +160,35 @@ def test_transfer_ownership(click_runner, testerchain, agency_local_registry):
...
+    # Test transfer ownersh
+
+
+def test_transfer_ownership_staking_interface_router(click_runner, testerchain, agency_local_registry):
+
+    maclane = testerchain.unassigned_accounts[0]
+
+    ownership_command = ('transfer-ownership',
+                         '--registry-infile', agency_local_registry.filepath,
+                         '--contract-name', StakingEscrowDeployer.contract_name,
```
Comment (derekpierre) [inline tests/acceptance/cli/deploy/test_deploy_cli_commands.py:172]: Did you mean StakingInterfaceDeployer?

Comment (KPrasch) [reply tests/acceptance/cli/deploy/test_deploy_cli_commands.py]: :heavy_check_mark:

**Review by derekpierre — APPROVED** [review]
🎸

**Review by cygnusv — APPROVED** [review]
Thanks for the fix, @KPrasch ! 
I took the liberty of fixing the conflicts, rebasing, and adding a newsfragment.

**Pull request merged**